tests/e2e/tests/e2e_util/util.py

```diff
@@ -174,11 +175,10 @@ def wait_until_k8s_ready(h: harness.Harness, instances: Union[str, List[str]]):
         hostname = (
             h.exec(instance, ["hostname"], capture_output=True).stdout.decode().strip()
         )
-        result = retry_until_condition(
-            h,
-            instance,
-            ["k8s", "kubectl", "get", "node", hostname, "--no-headers"],
-            condition=lambda p: "Ready" in p.stdout.decode(),
+        result = (
+            stubbornly(retries=15, delay_s=5)
+            .until(lambda p: "Ready" in p.stdout.decode())
+            .exec(f"k8s kubectl get node {hostname} --no-headers")
```

Comment:
i think this was missed

Reply:
thanks, got it.  i was letting the tests find them :D